### ISSUE TITLE
Question: design files for models 13 and 14 in recent Benchmarking Paper

### ISSUE THREAD

**Issue by jdkent** [PennBBL/xcpEngine#94]
Hello,

Awesome work on this pipeline! I am curious about the order of operations for models 13 and 14 in your recent paper: "Benchmarking of participant-level confound regression strategies for the control of motion artifact in studies of functional connectivity".

> The first de-noising step occurs immediately after classification. All component time series (signal and noise) are included as predictors in the linear model, and the residual BOLD time series is obtained via partial regression of only the noise time series. A second confound regression step occurs after temporal filtering, wherein mean signals from WM and CSF (models 13 and 14) and the global signal (model 14 only) were regressed from the data.

While I feel as though it is implied through the wording, I wanted to make sure the mean signals from WM, CSF, and global are extracted from the residual timeseries after partial regression and temporal filtering, correct?

Thanks!
James

**Comment by rciric** [comment]
Hello James,

That's right -- we followed our understanding of the [Pruim paper](http://www.sciencedirect.com/science/article/pii/S1053811915001822) in this case: "Similarly, as signal from WM and CSF time-series might also contain variance related to motion artifacts, we applied ICA-AROMA prior to nuisance regression. . . We implemented the removal of residual noise by means of WM and CSF regression."

As a word of caution, we're currently in a phase of rapid development, and the code in the pipeline's `master` branch isn't supported -- it was built specifically to run on our cluster and to process PNC data, and it's quite a mess as a result. The ICA-AROMA implementation in the `master` branch consequently hasn't been stably maintained, so I'd recommend using the [official code](https://github.com/rhr-pruim/ICA-AROMA) released by the Pruim team. I also suspect CPAC might support something similar.

Alternatively, the new version of our pipeline set for release a few days from now (v0.6.0, currently viewable in the `ACCELERATOR` branch) will stably support ICA-AROMA and will also automatically report group-level QC-FC measures. We're also planning to distribute design files corresponding to the 14 evaluated pipelines (as well as pipelines evaluated in the more recent Parkes paper) to go with that release and hopefully Dockerise within the month to support reproduction. In fact, v0.6.0-conformed design files analogous to models [13](https://github.com/PennBBL/xcpEngine/blob/ACCELERATOR/designs/fc-aroma.dsn) and [14](https://github.com/PennBBL/xcpEngine/blob/ACCELERATOR/designs/fc-aroma%2Bgsr.dsn) are now available in the `ACCELERATOR` branch.

Let me know if there is anything else that I can do to clarify!

Best,
Rastko
